packages/plugins/i18n/admin/src/components/LocaleListCell.tsx

```diff
@@ -60,7 +60,7 @@ const LocaleListCell = ({
         })}
       >
         <Button type="button" onClick={handleTogglePopover} ref={buttonRef}>
-          <ActionWrapper alignItems="center" justifyContent="center" height="2rem" width="2rem">
+          <ActionWrapper alignItems="center" justifyContent="center" height="2rem" width="10rem">
```

Comment:
Do we need the width even declared here? below we have a `maxWidth` set on the typography with ellipse, so the cell will just fit the content like we want without a set value?

Reply:
Yes, you are right, we can use the maxWidth instead of the wrapper width